components/collator/src/comparison.rs

```diff
@@ -126,12 +126,12 @@ impl Collator {
             }
             filtered_locale
         };
-        let resource_options: ResourceOptions = locale.into();
+        let data_options: DataOptions = locale.into();
```

Comment:
Nit (optional): Since you're in this code anyway would you consider changing it to DataLocale now? If not or you still want to talk about that, it can be done later

Reply:
These are the names we've agreed on for now, let's do this separately. DataLocale also requires renaming lots of variables from `options` to `locale`, so it's actually more than just renaming the type.